src/controllers/verifyCommand.ts

```diff
@@ -22,12 +22,15 @@ export async function verifyCommand(
     discriminator,
     env
   );
-  if (response?.status === 201) {
+  if (response?.status === 201 || response?.status === 200) {
     const verificationSiteURL = config(env).VERIFICATION_SITE_URL;
     const message = `${verificationSiteURL}/discord?token=${token}`;
-    await sendDiscordDm(userId, env, message);
-    return discordTextResponse("Please check the DM");
+    const ephemeralResponse = await discordephemeralResponse(
+      message,
+      env.DISCORD_TOKEN
+    );
+    return ephemeralResponse;
   } else {
-    return discordTextResponse("Error, please use the verify command again. ");
+    return discordephemeralResponse(RETRY_COMMAND, env.DISCORD_TOKEN);
```

Comment:
do we need to pass `DISCORD_TOKEN` in arg, we could directly access env.DISCORD_TOKEN inside the func

Reply:
Good point

Reply:
I tried using `process.env.DISCORD_TOKEN` but I am receiving as `undefined` token. 
It is only working for `env.DISCORD_TOKEN` where `env` is parameter with type `env` defined in `default.types.d.ts`  

The problem of chaining and passing env is still unresolved, that can be discussed by creating a separate issue ticket.